src/site/paragraphs/expandable_text.drupal.liquid

```diff
@@ -0,0 +1,21 @@
+{% comment %}
+Example data:
+    "entity": {
+    "entityId": "3887",
+    "entityBundle": "expandable_text",
+    "fieldWysiwyg": {
+    "processed": "<p>Test Test</p>"
+    },
+    "fieldTextExpander": "Test Expander"
+    }
+{% endcomment %}
+
+{% if entity.fieldTextExpander %}
+    <div class="form-expanding-group borderless-alert additional-info-container">
+        <div class="additional-info-title">{{ entity.fieldTextExpander }}</div>
+
+        {% if entity.fieldWysiwyg %}
+            <div class="additional-info-content usa-alert-text" hidden>{{ entity.fieldWysiwyg.processed }}</div>
+        {% endif %}
+    </div>
+{% endif %}
```

Comment:
I think this template could be shared as well, if we incorporate an `analyticsId` field. Maybe we can include this in `src/site/blocks/alert.drupal.liquid`? (And in `src/site/paragraphs/alert.drupal.liquid` as well, if we can't consolidate that template with `src/site/blocks/alert.drupal.liquid`)

Reply:
the analyticsId is tied to the alert and doesn't show up in the separate expander paragraph unfortunately. There wouldn't be a way to include it here without making a backend change in the CMS.